content/20.zksync-protocol/20.contracts/10.l1-contracts/20.shared-bridges.md

```diff
@@ -183,14 +183,14 @@ In this section, we will present some diagrams showing the interaction of differ
 
 #### New Chain
 
-A chain registers in the Bridgehub, this is where the chain ID is determined. The chain’s governor specifies the State
-Transition that they plan to use. In the first version only a single State Transition contract will be available for
+A chain registers in the Bridgehub, this is where the chain ID is determined. The chain’s governor specifies the Chain Type
```

Comment:
FYI here the rename makes sense, as indeed the admin selects the type of the chain they want to use